sandbox/grommet-app/src/pages/layouts/kinds/OpsRamp/ResourceAlerts.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Box, Text } from 'grommet';
+import { Metric, metricSizes } from '../../../../components';
+import {
+  StatusGoodSmall,
+  StatusCriticalSmall,
+  StatusWarningSmall,
+  StatusInfo,
+  StatusPlaceholderSmall,
+} from 'grommet-icons';
+
+interface AlertItemProps {
+  label: string;
+  Icon: React.ComponentType<{ color?: string }>;
+  color: string;
+  value: number;
+  size: string;
+}
+
+const AlertItem: React.FC<AlertItemProps> = ({
+  label,
+  Icon,
+  color,
+  value,
+  size,
+}) => (
+  <AlertMetric
+    size={size}
+    label={label}
+    value={
+      <Box direction="row" align="center" gap="xsmall">
+        <Icon color={color} />
+        <Text size={metricSizes[size].value} color="text-strong" weight={500}>
+          {value}
+        </Text>
+      </Box>
+    }
+  />
+);
+
+interface AlertMetricProps {
+  label: string;
+  size: string;
+  value: JSX.Element;
+}
+
+const AlertMetric: React.FC<AlertMetricProps> = ({ label, size, value }) => (
+  <Metric
+    align="center"
+    label={label}
+    value={value}
+    unit={null}
+    options={{}}
+    reverse
+    size={size}
+  />
+);
+
+const TicketNotesSummary: React.FC = () => (
+  <Box pad={{ vertical: 'small' }} direction="row" gap="small">
+    <Text>
+      Tickets: <Text weight="bold">1</Text>
```

Comment:
In their design, it looks like these numbers would actually be anchors that would take me to tickets, etc.